btrack/utils.py

```diff
@@ -221,6 +226,86 @@ def nans_idx(x):
     return data, properties, graph
 
 
+def napari_to_tracks(
+    data: npt.NDArray,
+    properties: Optional[dict[str, npt.ArrayLike]],
+    graph: Optional[dict[int, list[int]]],
+) -> list[btypes.Tracklet]:
+    """Convert napari Tracks to a list of Tracklets.
+
+    Parameters
+    ----------
+    data : array (N, D+1)
+        Coordinates for N points in D+1 dimensions. ID,T,(Z),Y,X. The first
+        axis is the integer ID of the track. D is either 3 or 4 for planar
+        or volumetric timeseries respectively.
+    properties : dict {str: array (N,)}
+        Properties for each point. Each property should be an array of length N,
+        where N is the number of points.
+    graph : dict {int: list}
+        Graph representing associations between tracks. Dictionary defines the
+        mapping between a track ID and the parents of the track. This can be
+        one (the track has one parent, and the parent has >=1 child) in the
+        case of track splitting, or more than one (the track has multiple
+        parents, but only one child) in the case of track merging.
+
+    Returns
+    -------
+    tracks : list[btypes.Tracklet]
+        A list of tracklet objects from BayesianTracker.
+
+    """
+
+    if data.shape[1] == Dimensionality.FIVE:
+        track_id, t, z, y, x = data.T
+    elif data.shape[1] == Dimensionality.FOUR:
+        track_id, t, y, x = data.T
+        z = np.zeros_like(x)
+    else:
+        raise ValueError(
+            "Data must have either 4 (ID, t, y, x) or 5 (ID, t, z, y, x) columns, "
+            f"not {data.shape[1]}"
+        )
+
+    # Create all PyTrackObjects
+    objects_dict = {
+        "ID": properties.get("refs", np.arange(track_id.size)),
+        "t": t,
+        "x": x,
+        "y": y,
+        "z": z,
+        "dummy": properties.get("dummy", np.full_like(track_id, fill_value=False)),
+        "label": properties.get("state", np.full_like(track_id, fill_value=5)),
```

Comment:
@quantumjot I wasn't sure what the default `label` of the `PyTrackObjects` should be, but I used `5` as this was the value for all labels in the sample dataset. Is that reasonable?

Reply:
5 should be fine, but just to be sure, I would use this enum instead:

https://github.com/quantumjot/btrack/blob/99301cc479b9bee0c01e8823443e64899dcb3ddb/btrack/constants.py#L71-L79

`States.NULL` has a value of 5

Reply:
ah perfect, thanks! I wasn't sure where those values were coming from